app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/masterpasswordguidance/MasterPasswordGuidanceScreen.kt

```diff
@@ -0,0 +1,243 @@
+                    }
+                },
+            )
+        },
+    ) { innerPadding ->
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .verticalScroll(rememberScrollState())
+                .padding(innerPadding)
+                .padding(horizontal = 16.dp),
+            verticalArrangement = Arrangement.spacedBy(16.dp),
+        ) {
+            Column(
+                Modifier
+                    .fillMaxWidth()
+                    .clip(RoundedCornerShape(size = 4.dp))
+                    .background(MaterialTheme.colorScheme.surfaceContainerLowest),
+            ) {
+                Column(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(all = 24.dp),
+                    verticalArrangement = Arrangement.spacedBy(8.dp),
+                ) {
+
+                    Text(
+                        text = stringResource(R.string.what_makes_a_password_strong),
+                        style = MaterialTheme.typography.titleMedium,
+                        color = MaterialTheme.colorScheme.onSurface,
+                    )
+                    Text(
+                        style = MaterialTheme.typography.bodyMedium,
+                        color = MaterialTheme.colorScheme.onSurfaceVariant,
+                        text = stringResource(
+                            R.string.the_longer_your_password_the_more_difficult_to_hack,
+                        ),
+                    )
+                }
+                HorizontalDivider()
+                Column(
+                    Modifier
+                        .fillMaxWidth()
+                        .padding(horizontal = 24.dp, vertical = 16.dp),
+                ) {
+                    Text(
+                        text = stringResource(R.string.the_strongest_passwords_are_usually),
+                        style = MaterialTheme.typography.titleSmall,
+                        color = MaterialTheme.colorScheme.onSurface,
+                    )
+                    Spacer(modifier = Modifier.height(8.dp))
+                    BulletTextRow(text = stringResource(R.string.twelve_or_more_characters))
+                    BulletTextRow(
+                        text = stringResource(
+                            R.string.random_and_complex_using_numbers_and_special_characters,
+                        ),
+                    )
+                    BulletTextRow(
+                        text = stringResource(R.string.totally_different_from_your_other_passwords),
+                    )
+                }
+            }
+            TryGeneratorCard(
+                onCardClicked = remember(viewModel) {
+                    {
+                        viewModel.trySendAction(
+                            MasterPasswordGuidanceAction.TryPasswordGeneratorAction,
+                        )
+                    }
+                },
+            )
+        }
```

Comment:
Be sure to add a `Spacer` with navigation bars padding here.

Reply:
would adding to the API of the scaffold be an issue, guessing there is cases where we don't want this? but maybe it could be a arg: `addNavBarPadding: Boolean = false,`

Reply:
Discussed this offline, there are issues with content scrolling behind the navbar with this approach. 😭